Samsung/A50/res/values/notch.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+	<dimen name="rounded_corner_content_padding">64px</dimen>
+	<dimen name="rounded_corner_radius">92px</dimen>
```

Comment:
This device has a real notch, don't just specify the corners :(

Reply:
Meh, worked well enough. I'll take a look tomorrow on other devices, then calculate it and update the PR.